src/types/bitcoin-core.d.ts

```diff
@@ -0,0 +1,1099 @@
+/* eslint-disable tsdoc/syntax */
+declare module 'bitcoin-core' {
```

Comment:
Pulling in this typedef is kinda crazy. It looks like this lib is hostile towards typescript https://github.com/ruimarinho/bitcoin-core/issues?q=typescript

Would you be open to using a different one like https://github.com/vansergen/rpc-bitcoin ?

Reply:
Great suggestion - I looked around for a sec but didn't see that package. Much better. I've implemented it.